doc/sphinx/installation.rst

```diff
@@ -84,9 +84,30 @@ are required to be able to compile and use |es|:
     Python
         |es|'s main user interface relies on Python 3.
 
+        We strongly recommend using Python environments to isolate
+        packages required by |es| from packages installed system-wide.
+        This can be achieved using venv [7]_, conda [8]_, or any similar tool.
+        Inside an environment, commands of the form
+        ``sudo apt install python3-numpy python3-scipy``
+        can be rewritten as ``python3 -m pip install numpy scipy``,
+        and thus do not require root privileges.
+
+        Depending on your needs, you may choose to install all |es|
+        dependencies inside the environment, or only the subset of
+        dependencies not already satisfied by your workstation or cluster.
+        For the exact syntax to create and configure an environment,
+        please refer to the tool documentation.
```

Comment:
More details and a "getting started with venv and conda" are now available on the GitHub wiki, under section [Tooling/Environments](https://github.com/espressomd/espresso/wiki/Tooling#environments).